src/rolch/methods/factory.py

```diff
@@ -16,6 +17,9 @@ def from_string(method: str):
             return LassoPathMethod()
         elif method == "ols":
             return OrdinaryLeastSquaresMethod()
+        elif method == "elasticnet":
+            print("Setting default alpha to 0.5.")
+            raise ElasticNetPathMethod(alpha=0.5)
```

Comment:
This should be `return`, not `raise`.
```suggestion
            return ElasticNetPathMethod(alpha=0.5)
```